src/components/Grid/index.js

```diff
@@ -0,0 +1,11 @@
+import { Grid, Row, Col } from './GridComponents'
+
+Grid.Row = Row
+Grid.Col = Col
+
+export default Grid
```

Comment:
I understand that this is similar way to export as in bootstrap, but is this really necessary to export `Grid` as default? Because I don't any use of this, so maybe better will be to do like this:

```JavaScript
export {
  Grid,
  Row,
  Col,
} from './GridComponents'
```
?

Reply:
Makes sense.